crates/storage/engines/libmdbx.rs

```diff
@@ -337,6 +339,17 @@ impl StoreEngine for Store {
                 .map_err(|_| StoreError::DecodeError),
         }
     }
+
+    fn world_state(&self, block_number: BlockNumber) -> Result<Option<Trie>, StoreError> {
```

Comment:
I would rename to `state_trie`. I haven't seen references to "world" in other client implementations, just state

Reply:
It is referred to as World State in the yellow paper, but I agree, we can just name it state trie